### PR TITLE
Fix tests that produce legacy block for maps

### PR DESCRIPTION
**Pull request by haozhun**

### CODE REVIEW

File: presto-main/src/test/java/com/facebook/presto/type/TestMapOperators.java [hunk 1]
```diff
@@ -91,33 +83,6 @@ public static Slice uncheckedToJson(@SqlType("varchar(x)") Slice slice)
     }
 
     @Test
-    public void testStackRepresentation()
```
Comment (raghavsethi) [inline presto-main/src/test/java/com/facebook/presto/type/TestMapOperators.java]: Why was this removed?

Comment (haozhun) [inline presto-main/src/test/java/com/facebook/presto/type/TestMapOperators.java]: This test hasn't been doing anything reasonable/useful for a while.
